src/components/RichEditor/RichEditorWrapper.test.tsx

```
@@ -0,0 +1,280 @@
+
+    it('deve mostrar todas as dependências necessárias no comando', async () => {
+      mockShouldThrowError = true;
+      mockErrorMessage = '@tiptap error';
+
+      render(<RichEditor />);
+
+      await waitFor(() => {
+        const yarnCommand = screen.getByText(/yarn add/);
+        expect(yarnCommand.textContent).toContain('@tiptap/core');
+        expect(yarnCommand.textContent).toContain('@tiptap/starter-kit');
+        expect(yarnCommand.textContent).toContain('@tiptap/extension-underline');
```

ESLint

[error] Replace `'@tiptap/extension-underline'` with `⏎··········'@tiptap/extension-underline'⏎········`

(prettier/prettier)